vms/platformvm/docs/mempool_gossiping.md

```diff
@@ -1,16 +0,0 @@
-# Mempool Gossiping
```

Comment:
gossiping has been consolidated into an sdk, shared across VMs. I feel we should document its use in the sdk.

Reply:
SGTM, could we link to the gossip package (even if there is no README there at this time) from the technical content section in the README?